workflow.txt

```diff
@@ -1,10 +1,10 @@
+fork on github
 git clone
-git checkout -b my-solution
+git add .
```

Comment:
before we add and commit files, we need to switch to a different branch
`git checkout -b my-solution`